src/components/GenericWarningModal.jsx

```diff
@@ -54,7 +54,6 @@ const GenericWarningModal = ({
 GenericWarningModal.propTypes = {
   displayTitle: PropTypes.string.isRequired,
   displayText: PropTypes.string.isRequired,
-  displayImage: PropTypes.string,
```

Comment:
Same as above

Reply:
i removed it as it's unused - there's a similar `displayImg` prop up above though so actually i'll add this `PropType` back in + change the `displayImg` to `displayImage`